src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskProviderTest.java

```diff
@@ -224,6 +225,7 @@ private Task createTaskWithWarnings() {
             new WarningValues(warnings),
             "Case Management Category",
             "hearing_work",
+            "Hearing work",
```

Comment:
could we also update the consumer (TaskManagerGetTaskConsumerTest) to now include the workType label